DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/RecoContainerCreateTracksVariadic.h

```diff
@@ -302,6 +304,29 @@ void o2::globaltracking::RecoContainer::createTracksVariadic(T creator, GTrackID
     }
   }
 
+  // ITS-TOF matches
+  {
+    currentSource = GTrackID::ITSTOF;
+    if (srcSel[currentSource]) {
+      if (matchesITSTOF.size() && (!tofClusters.size() || !tracksITS.size())) {
+        throw std::runtime_error(fmt::format("Global-TOF tracks ({}) require ITS-TPC tracks ({}) and TOF clusters ({})",
```

Comment:
please fix the comment: " require ITS tracks ..."

Reply:
done